stdlib/exn.mli

```diff
@@ -0,0 +1,100 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           *)
+(*                                                                        *)
+(*   Copyright 2018 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Exception values.
+
+    Facilities for printing exceptions and obtaining traces.
+
+    @since 4.08 *)
+
+type t = exn = ..
+(** The type for exceptions. *)
+
+val to_string: t -> string
+(** [to_string e] returns a string representation of the exception [e]. *)
+
+val set_tracing: bool -> unit
+(** [set_recording_status b] turns recording of exception traces on (if [b
+    = true]) or off (if [b = false]).  Initially, traces are not recorded,
+    unless the [b] flag is given to the program through the [OCAMLRUNPARAM]
+    variable. *)
+
+val tracing: unit -> bool
+(** Returns [true] if exception traces are currently recorded, [false] if
+    not. *)
+
+val last_trace: unit -> Stack_trace.t
+(** Returns a stack trace containing the program locations where the
+    most-recently raised exception was raised and where it was propagated
+    through function calls.
+
+    If the call is not inside an exception handler, the returned trace is
+    unspecified. If the call is after some exception-catching code (before in
+    the handler, or in a when-guard during the matching of the exception
+    handler), the stack trace may correspond to a later exception than the
+    handled one. *)
+
+external raise: t -> 'a = "%raise"
```

Comment:
These functions are missing a doc string. I would move them before the tracing one and under their own `{1:raising Raising}` heading.

Reply:
I added some doc and put them under their own heading.